node/src/command.rs

```diff
@@ -23,12 +23,6 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	Ok(match id {
 		#[cfg(feature = "neumann-node")]
 		"dev" => Box::new(chain_spec::neumann::development_config()),
-		#[cfg(feature = "neumann-node")]
-		"" | "local" => Box::new(chain_spec::neumann::local_testnet_config()),
```

Comment:
do you want to change the `""` string to be turing-dev or just "dev"?

Reply:
do you mean nuemann-dev?

Reply:
this one.
`"dev" => Box::new(chain_spec::neumann::development_config()),`
doesn't look like there is a `neumann-dev`

Reply:
Sorry, that still came out confusing.
On L27 here you've deleted the default option when someone doesn't specify a chain (`"" | "local"`). I was questioning whether there should be a new default.